benchmarking/README.md

```diff
@@ -0,0 +1,57 @@
+# Benchmarking
+
+[Contact me](mailto:rli@olin.edu) with questions! I'll try to explain everything here but I've probably missed something.
+
+The benchmarking folder is composed of 4 scripts. They were each designed to benchmark some aspect of the Invisible Map workflow and debug our code. Each script should explain (briefly) what it does at the top, but here is all of the explanations in one place.
+
+Here's a brief overview of the organization of this repository:
+- `/datasets/` contains jsons that are used for testing in some way or another. inside this folder, the `sweeped` folder contains the processed results after a sweep. 
+- `/debugging_txts/` are txt files of terminal output that helped us both debug and compare results of our tests.
+- `/imgs/` are images automatically generated by visualizations. 
+- `benchmarking_config.json` is the configuration file of this repository. Our testing configurations are already added to it, but feel free to add more files should you want to test further1
+
+## Oblique v.s. Straight Tag Comparisons
+---
+
+`rotational_diff_visualizer.py`
+
+One of the interesting things that we noticed purely experimentally is that our algorithm worked better when tags were observed straight on instead of obliquely. In order to get more information about why that was the case, we wanted to see how the information that Invisible Maps collects changes based upon the type of observation.
```

Comment:
It would be awesome to include plots here (as images).  This will also help you all if you want to show this off when applying to jobs and the like.